src/bindings/python/tests/test_graph/test_pooling.py

```diff
@@ -236,3 +237,83 @@ def test_max_pool(op_params, input_shape, expected_output_shape, op_name):
     assert list(node.get_output_shape(1)) == expected_output_shape
     assert node.get_output_element_type(0) == Type.f32
     assert node.get_output_element_type(1) == Type.i32
+
+
+avg_pooling_opset16_test_params = [
+    (
+        [
+            [2, 2],  # strides
+            [0, 0],  # pads_begin
+            [0, 0],  # pads_end
+            [2, 2],  # kernel_shape
+            True,  # exclude_pad
+            "floor",  # rounding_type
+            None,  # auto_pad
+            [1, 1],  # dilations
+        ],
+        [1, 1, 4, 4],  # input_shape
+        [1, 1, 2, 2],  # expected_output_shape
+    ),
+    (
+        [
+            [1, 1],  # strides
+            [0, 0],  # pads_begin
+            [0, 0],  # pads_end
+            [2, 2],  # kernel_shape
+            False,  # exclude_pad
+            "floor",  # rounding_type
+            None,  # auto_pad
+            [2, 2],  # dilations
+        ],
+        [1, 1, 5, 5],  # input_shape
+        [1, 1, 3, 3],  # expected_output_shape
+    ),
+    (
+        [
+            [2, 2],  # strides
+            [1, 1],  # pads_begin
+            [1, 1],  # pads_end
+            [2, 2],  # kernel_shape
+            False,  # exclude_pad
+            "floor",  # rounding_type
+            None,  # auto_pad
+            [2, 2],  # dilations
+        ],
+        [1, 3, 10, 10],  # input_shape
+        [1, 3, 5, 5],  # expected_output_shape
+    ),
+]
+
+
+@pytest.mark.parametrize(
+    ("op_params", "input_shape", "expected_output_shape"),
+    avg_pooling_opset16_test_params,
+)
+@pytest.mark.parametrize("op_name", ["avg_pool", "avgPool", "avgPoolOpset16"])
+def test_avg_pool_opset16_with_dilations(op_params, input_shape, expected_output_shape, op_name):
+    param = ov.parameter(input_shape, name="A", dtype=np.float32)
+    node = ov16.avg_pool(param, *op_params, name=op_name)
+    assert node.get_type_name() == "AvgPool"
+    assert node.get_friendly_name() == op_name
+    assert node.get_output_size() == 1
+    assert list(node.get_output_shape(0)) == expected_output_shape
+    assert node.get_output_element_type(0) == Type.f32
+
+
+def test_avg_pool_opset16_without_dilations():
+    param = ov.parameter([1, 1, 4, 4], name="A", dtype=np.float32)
+    node = ov16.avg_pool(
+        param,
+        strides=[2, 2],
+        pads_begin=[0, 0],
+        pads_end=[0, 0],
+        kernel_shape=[2, 2],
+        exclude_pad=True,
+        rounding_type="floor",
+        name="avg_pool_no_dilations"
+    )
```

Comment:
Can at least one test case with different than default "floor" for "rounding_type" be added to ensure the attr is correctly passed?
Or even easier, the opset16 no-dilations test (test_avg_pool_opset16_without_dilations) can be parametrized with already existing above for previous opset `avg_pooling_test_params` to ensure backward compatibility.

https://github.com/p-wysocki/openvino/blob/de1cc529624d038334599be88d3271efc9f017fb/src/bindings/python/tests/test_graph/test_pooling.py#L13-L34